bedrock/firefox/views.py

```diff
@@ -265,7 +265,8 @@ def all_downloads(request, platform, channel):
 
     context = {
         'platform': platform,
-        'platforms': product.platforms(channel),
+        'platforms': product.platforms(channel, True),
+        'platform_cls': product.platform_classification,
```

Comment:
While this works for `FirefoxAndroid` (I believe through the `__getattr__` implementation in `ProductDetails`), I think it would be clearer to explicitly set `plaform_classification = None` (with a comment) in the `FirefoxAndroid` class in `firefox_details.py`. 

This will make understanding/reading the code quite a bit simpler for those unfamiliar with `ProductDetails`.